.github/workflows/main-branches.yml

```
@@ -2,16 +2,32 @@ name: Mirror
     branches:
-      - main
 
 env:
   MIRROR_URL: "git@github.com:EpitechPromo2028/B-OOP-400-BDX-4-1-raytracer-nolann.dubos.git"
   EXECUTABLES: "raytracer"
 
 jobs:
+  check_compilation:
+    name: Check compilation
+    runs-on: ubuntu-latest
+    container: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)